app/ios/MemexShare/MemexShare.entitlements

```diff
@@ -8,7 +8,7 @@
 	</array>
 	<key>keychain-access-groups</key>
 	<array>
-		<string>$(AppIdentifierPrefix)group.io.worldbrain.memex</string>
+		<string>5YUPQC9D96.group.io.worldbrain.memex</string>
```

Comment:
@blackforestboi why did this and the main app's entitlement need to change?

Reply:
This was one of the devs that worked on it. I think you can reverse it.